config/auto_settings.default.json

```diff
@@ -1,224 +1,136 @@
-            {
-              "max_diff": 0
-            }
-          ],
-          [
-            "set_max_ping_autokick",
-            {
-              "max_ms": 700
-            }
-          ],
-          [
-            "set_team_switch_cooldown",
-            {
-              "minutes": 5
-            }
-          ]
-        ]
-      ],
-      [
-        "between",
-        [20, 50],
-        [
-          [
-            "set_idle_autokick_time",
-            {
-              "minutes": 120
-            }
-          ],
-          [
-            "set_autobalance_threshold",
-            {
-              "max_diff": 1
-            }
-          ],
-          [
-            "set_max_ping_autokick",
-            {
-              "max_ms": 600
-            }
-          ],
-          [
-            "set_team_switch_cooldown",
-            {
-              "minutes": 10
-            }
-          ]
-        ]
-      ],
-      [
-        "between",
-        [50, 70],
-        [
-          [
-            "set_idle_autokick_time",
-            {
-              "minutes": 60
-            }
-          ],
-          [
-            "set_autobalance_threshold",
-            {
-              "max_diff": 2
-            }
-          ],
-          [
-            "set_max_ping_autokick",
-            {
-              "max_ms": 500
-            }
-          ],
-          [
-            "set_team_switch_cooldown",
-            {
-              "minutes": 15
-            }
-          ]
-        ]
-      ],
-      [
-        "between",
-        [70, 96],
-        [
-          [
-            "set_idle_autokick_time",
-            {
-              "minutes": 20
-            }
-          ],
-          [
-            "set_autobalance_threshold",
-            {
-              "max_diff": 2
-            }
-          ],
-          [
-            "set_max_ping_autokick",
-            {
-              "max_ms": 500
-            }
-          ],
-          [
-            "set_team_switch_cooldown",
-            {
-              "minutes": 30
-            }
-          ],
-          [
-            "set_vip_slots_num",
-            {
-              "num": 0
-            }
-          ]
-        ]
-      ],
-      [
-        "between",
-        [96, 100],
-        [
-          [
-            "set_idle_autokick_time",
-            {
-              "minutes": 10
-            }
-          ],
-          [
-            "set_autobalance_threshold",
-            {
-              "max_diff": 0
-            }
-          ],
-          [
-            "set_max_ping_autokick",
-            {
-              "max_ms": 500
-            }
-          ],
-          [
-            "set_team_switch_cooldown",
-            {
-              "minutes": 30
-            }
-          ],
-          [
-            "set_vip_slots_num",
-            {
-              "num": 1
-            }
-          ]
-        ]
-      ]
-    ]
+      }
+    }
+  ],
+  "_available_commands": {
```

Comment:
oh damn you typed all that.
We can generate that stuff automatically just like I do for the api endpoints